### PR TITLE
Getting the solution to load without freezing + updates

### PR DESCRIPTION
**Pull request by mattleibow**
### Description of Change ###

Just fixing up the solution to make it better for dev. Also updating the Xamarin.Forms, xunit and Material. This mainly involved re-creating the projects and copying the .csproj files. Seems something went wonky over time.

### Bugs Fixed ###

None.

### API Changes ###

None.

### Behavioral Changes ###

Describe any non-bug related behavioral changes that may change how users app behaves when upgrading to this version of the codebase.

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Has samples (if omitted, state reason in description)
- [ ] Rebased on top of master at time of PR
- [ ] Changes adhere to coding standard
- [ ] Updated documentation ([see walkthrough](https://github.com/xamarin/Essentials/wiki/Documenting-your-code-with-mdoc))

### CODE REVIEW

File: DeviceTests/DeviceTests.iOS/DeviceTests.iOS.csproj [hunk 1]
```diff
@@ -55,6 +46,7 @@
     <DefineConstants>DEBUG</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <MtouchLink>SdkOnly</MtouchLink>
```
Comment (Redth) [inline DeviceTests/DeviceTests.iOS/DeviceTests.iOS.csproj]: We may want link all here to test out Our linked on tests

Comment (mattleibow) [reply DeviceTests/DeviceTests.iOS/DeviceTests.iOS.csproj]: I see the device tests builds iOS as release, but Android and UWP as debug...

Comment (Redth) [reply DeviceTests/DeviceTests.iOS/DeviceTests.iOS.csproj]: Looks like this was the existing implicit default, so I guess it's ok!

File: Xamarin.Essentials/Xamarin.Essentials.csproj [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
   <PropertyGroup>
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">netstandard1.0;netstandard2.0;Xamarin.iOS10;Xamarin.TVOS10;Xamarin.WatchOS10;MonoAndroid60;MonoAndroid70;MonoAndroid71;MonoAndroid80;MonoAndroid81;MonoAndroid90;uap10.0.16299;tizen40;</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard1.0;netstandard2.0;Xamarin.iOS10;Xamarin.TVOS10;Xamarin.WatchOS10;MonoAndroid60;MonoAndroid70;MonoAndroid71;MonoAndroid80;MonoAndroid81;MonoAndroid90;tizen40;</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">netstandard2.0;Xamarin.iOS10;Xamarin.TVOS10;Xamarin.WatchOS10;MonoAndroid81;MonoAndroid90;uap10.0.16299;tizen40;</TargetFrameworks>
```
Comment (mattleibow) [inline Xamarin.Essentials/Xamarin.Essentials.csproj]: This is probable the most important change here

**Pull request merged**